livekit-rtc/livekit/rtc/participant.py

```diff
@@ -88,13 +91,14 @@ def __init__(
     ) -> None:
         super().__init__(owned_info)
         self._room_queue = room_queue
-        self.tracks: dict[str, LocalTrackPublication] = {}  # type: ignore
+        self.track_publications: dict[str, LocalTrackPublication] = {}  # type: ignore
 
     async def publish_data(
         self,
         payload: Union[bytes, str],
```

Comment:
should payload be positional?

Reply:
I like it positional because it's the only value with no default so it's really convenient